test/integration/ethereum/optimisticAuctionRebalanceExtenisonV1.spec.ts

```diff
@@ -412,7 +414,7 @@ if (process.env.INTEGRATIONTEST) {
                       {
                         collateral: collateralAssetAddress,
                         liveness,
-                        bondAmount: BigNumber.from(0),
```

Comment:
@pblivin0x Normally this should work with 0 bondAmount as well as we are comparing it against the minimumBond as returned by the oracle for a given collateral token.
However it seems that the oracle returns a minimum bond of "0" for INDEX which we should raise / double check with the UMA team. 

See: 
https://github.com/IndexCoop/index-coop-smart-contracts/blob/9e2082d01efb510c6ecf9228c5ad324d57134149/contracts/adapters/OptimisticAuctionRebalanceExtensionV1.sol#L405